src/main/resources/templates/thymeleaf/absences/absences-overview.html

```diff
@@ -0,0 +1,396 @@
+            </th>
+            <th
+              scope="row"
+              class="tw-py-0.5 tw-pl-2 tw-pr-4 print:tw-py-1.5"
+            >
+              <div class="tw-flex tw-flex-col tw-justify-center tw-leading-tight">
+                <th:block th:text="${person.firstName}" />&nbsp;
+                <span th:text="${person.lastName}"></span>
+              </div>
+            </th>
+            <td th:each="absence : ${person.days}">
+              <div
+                class="cal-day"
+                th:classappend="|${absence.type != null && absence.type.absenceFull ? 'absence-full' : ''}${absence.type != null && absence.type.absenceMorning ? ' absence-morning' : ''}${absence.type != null && absence.type.absenceNoon ? ' absence-noon' : ''}${absence.type != null && absence.type.waitingVacationFull ? ' vacation-full-waiting' : ''}${absence.type != null && absence.type.waitingVacationMorning ? ' vacation-morning-waiting' : ''}${absence.type != null && absence.type.waitingVacationNoon ? ' vacation-noon-waiting' : ''}${absence.type != null && absence.type.allowedVacationFull ? ' vacation-full-approved' : ''}${absence.type != null && absence.type.allowedVacationMorning ? ' vacation-morning-approved' : ''}${absence.type != null && absence.type.allowedVacationNoon ? ' vacation-noon-approved' : ''}${absence.type != null && absence.type.sickNoteFull ? ' sick-note-full' : ''}${absence.type != null && absence.type.sickNoteMorning ? ' sick-note-morning' : ''}${absence.type != null && absence.type.sickNoteNoon ? ' sick-note-noon' : ''}${absence.type != null && absence.type.publicHolidayFull ? ' public-holiday-full' : ''}${absence.type != null && absence.type.publicHolidayMorning ? ' public-holiday-morning' : ''}${absence.type != null && absence.type.publicHolidayNoon ? ' public-holiday-noon' : ''}${absence.workday ? '' : ' no-workday'}|"
+              >
+                <div
+                  th:if="${not absence.workday}"
+                  class=""
+                  style="height: 100%; display: flex; align-items: center; justify-content: center"
+                >
+                  <svg th:replace="icon/ban::svg(className='no-workday-icon tw-w-5 tw-h-5 tw--translate-y-px tw-stroke-2')"></svg>
+                </div>
+                <span class="tw-hidden print:tw-inline print:tw-font-mono">
+                  <th:block
+                    th:if="${absence.type.absenceMorning}"
+                    th:text="#{absences.overview.absence.morning.abbr}">
+                  </th:block>
+                  <th:block
+                    th:if="${absence.type.absenceNoon}"
+                    th:text="#{absences.overview.absence.noon.abbr}"
+                  ></th:block>
+                  <th:block
+                    th:if="${absence.type.absenceFull}"
+                    th:text="#{absences.overview.absence.abbr}"
+                  ></th:block>
+                  <th:block
+                    th:if="${absence.type.waitingVacationMorning}"
```

Comment:
Das Muster ist unterschiedlich
waitingVacationMorning -> absences.overview.vacation.morning.abbr
allowedVacationMorning -> absences.overview.allowed.morning.abbr

vacation und allowed. Ggf sollte das oben waiting sein?

Reply:
Hm bei `absences.overview.vacation.morning.abbr` und `absences.overview.allowed.morning.abbr` ist das auch so. Ich schau mir das gleich mal genauer an.

Reply:
k, passt. Ist in den jsps auch so und die Druckansicht sieht auch gut aus.